website/content/api-docs/secret/pki.mdx

```diff
@@ -16,729 +16,735 @@ update your API calls accordingly.
-- [Delete Root](#delete-root)
-- [Sign Intermediate](#sign-intermediate)
-- [Sign Self-Issued](#sign-self-issued)
-- [Sign Certificate](#sign-certificate)
-- [Sign Verbatim](#sign-verbatim)
-- [Tidy](#tidy)
-- [Tidy Status](#tidy-status)
+- [Notice About New Multi-Issuer Functionality](#notice-about-new-multi-issuer-functionality)
+- [Issuing Certificates](#issuing-certificates)
+  - [List Roles](#list-roles)
+  - [Read Role](#read-role)
+  - [Generate Certificate and Key](#generate-certificate-and-key)
+  - [Sign Certificate](#sign-certificate)
+  - [Sign Intermediate](#sign-intermediate)
+  - [Sign Self-Issued](#sign-self-issued)
+  - [Sign Verbatim](#sign-verbatim)
+  - [Revoke Certificate](#revoke-certificate)
+- [Accessing Authority Information](#accessing-authority-information)
+  - [List Issuers](#list-issuers)
+  - [Read Issuer Certificate](#read-issuer-certificate)
+  - [Read Default Issuer Certificate Chain](#read-default-issuer-certificate-chain)
+  - [Read Issuer CRL](#read-issuer-crl)
+  - [List Certificates](#list-certificates)
+  - [Read Certificate](#read-certificate)
+- [Managing Keys and Issuers](#managing-keys-and-issuers)
+  - [List Issuers](#list-issuers)
+  - [List Keys](#list-keys)
+  - [Generate Root](#generate-root)
+  - [Generate Intermediate CSR](#generate-intermediate-csr)
+  - [Import CA Certificates and Keys](#import-ca-certificates-and-keys)
+  - [Read Issuer](#read-issuer)
+  - [Update Issuer](#update-issuer)
+  - [Delete Issuer](#delete-issuer)
+  - [Read Key](#read-key)
+  - [Update Key](#update-key)
+  - [Delete Key](#delete-key)
+  - [Delete All Issuers and Keys](#delete-all-issuers-and-keys)
+- [Managing Authority Information](#managing-authority-information)
+  - [List Roles](#list-roles)
+  - [Create/Update Role](#create-update-role)
+  - [Read Role](#read-role)
+  - [Delete Role](#delete-role)
+  - [Read URLs](#read-urls)
+  - [Set URLs](#set-urls)
+  - [Read Issuers Configuration](#read-issuers-configuration)
+  - [Set Issuers Configuration](#set-issuers-configuration)
+  - [Read Keys Configuration](#read-keys-configuration)
+  - [Set Keys Configuration](#set-keys-configuration)
+  - [Read CRL Configuration](#read-crl-configuration)
+  - [Set CRL Configuration](#set-crl-configuration)
+  - [Rotate CRLs](#rotate-crls)
+  - [Tidy](#tidy)
+  - [Tidy Status](#tidy-status)
+- [Cluster Scalability](#cluster-scalability)
 - [Managed Key](#managed-keys) (Enterprise Only)
+- [Vault CLI with DER/PEM responses](#vault-cli-with-der-pem-responses)
 
-## Read CA Certificate
+## Notice About New Multi-Issuer Functionality
 
-This endpoint retrieves the CA certificate _in raw DER-encoded form_. This is a
-bare endpoint that does not return a standard Vault data structure and cannot
-be read by the Vault CLI; use `/pki/cert` for that. If `/pem` is added to the
-endpoint, the CA certificate is returned in PEM format.
+Vault now allows a single PKI mount to have multiple Certificate Authority
```

Comment:
Not sure I like the temporal nature of the wording here and elsewhere... "now allows".  In 1.12 this will have already been the case.  I'd rather this be in the present tense "Vault allows" and lean on the release notes to introduce the newness.

Reply:
Oh, do you think this should live here, or should I put it in a release note or something?